environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -429,9 +429,17 @@ objects:
           - mountPath: /var/thanos/store
             name: data
             readOnly: false
-        volumes:
-        - emptyDir: {}
-          name: data
+        volumes: []
+    volumeClaimTemplates:
+    - metadata:
+        name: data
```

Comment:
Nit: Can we name this something more explicit than 'data'? Perhaps 'observatorium-thanos-store-data'

Reply:
it's already in an observatorium namespace, I think that would be duplicate, but `<statefulsetname>-data` would make sense I think

Reply:
Yeah `<statefulsetname>-data` sounds perfect. Just want to avoid the possibility of multiple components calling their PVC `data`

Reply:
Made the change upstream. Will bump here once merged upstream and it'll update without any other changes. :+1: 

https://github.com/metalmatze/kube-thanos/pull/33